neuralpredictors/data/transforms.py

```diff
@@ -298,24 +299,24 @@ def __init__(self, data, stats_source="all", exclude=None):
         itransforms[in_name] = lambda x: x * self._inputs_std + self._inputs_mean
 
         # -- responses
-        transforms["responses"] = lambda x: x * self._response_precision
-        itransforms["responses"] = lambda x: x / self._response_precision
+        transforms[out_name] = lambda x: x * self._response_precision
+        itransforms[out_name] = lambda x: x / self._response_precision
 
-        if "eye_position" in data.data_keys:
-            # -- eye position
-            self._eye_mean = np.array(
-                data.statistics["eye_position"][stats_source]["mean"]
-            )
-            self._eye_std = np.array(
-                data.statistics["eye_position"][stats_source]["std"]
-            )
-            transforms["eye_position"] = lambda x: (x - self._eye_mean) / self._eye_std
-            itransforms["eye_position"] = lambda x: x * self._eye_std + self._eye_mean
 
+        # -- behavior
+        transforms['behavior'] = lambda x: x
+
+        if "pupil_center" in data.data_keys:
+            self._eye_mean = np.array(data.statistics["pupil_center"][stats_source]["mean"])
+            self._eye_std = np.array(data.statistics["pupil_center"][stats_source]["std"])
+            transforms["pupil_center"] = lambda x: (x - self._eye_mean) / self._eye_std
+            itransforms["pupil_center"] = lambda x: x * self._eye_std + self._eye_mean
+
+        if "behavior" in data.data_keys:
             s = np.array(data.statistics["behavior"][stats_source]["std"])
 
             # TODO: same as above - consider other baselines
-            threshold = 0.01 * s.mean()
+            threshold = 0.01 * s
```

Comment:
this definition doesn't make sense as `s > 0.01 * s` is true all the time provided that `s` is a positive array (which it is as it's std values), so either completely get rid of this thresholding behavior, or come up with a meaningful thresholding scheme.